openlibrary/templates/lib/nav_head.html

```diff
@@ -15,13 +22,23 @@
   $     { "href": "/developers", "text": _("Developer Center") },
   $     { "href": "/help", "text": _("Help & Support") }
   $ ]
+  $ browseLinks = [
+  $     { "href": "/subjects", "text": _("Subjects") },
+  $     { "new": "", "href": "/explore", "text": _("Library Explorer") },
+  $     { "href": "/lists", "text": _("Lists") },
+  $     { "new": "", "href": "/collections", "text": _("Collections") },
+  $     { "href": "/k-12", "text": _("K-12 Student Library") },
+  $     { "href": "/random", "text": _("Random Book") },
+  $     { "href": "/advancedsearch", "text": _("Advanced Search") }
+  $ ]
   $ hamburgerProps = {
   $  'name': 'hamburger',
-  $  'links': loginLinks + moreLinks,
+  $  'links': [ { "subsection": _("My Open Library") } ] + loginLinks +
+  $     [ { "subsection": _("Browse") } ] + browseLinks + [ { "subsection": _("More") } ] + moreLinks,
```

Comment:
We don't want `subsection` to be part of this data structure; it makes this a little hard to read.

The app drawer shouldn't use `$:render_template("lib/header_dropdown"`; it's a different thing. We want something like this:

```html
$def render_app_drawer():
    <div class="mask-menu"></div>
    <div class="app-drawer">
        <h3>$_("My Open Library")</h3>
        $if ctx.user:
            $:render("lib/menu-list", loginLinks)
        $else:
            <div class="login-links">
                <a href="...">$_("Log in")</a>
                <a href="...">$_("Sign up")</a>
            </div>
        <h3>_$("Browser")</h3>
        $:render("lib/menu-list", browseList)
        ...
```

And then inside `<header>` we could have:

```html
    $:render_template("lib/header_dropdown", body=render_app_drawer())
```

Or something along those lines. That's not a blocker, but something we should do in a future PR to make this code easier to read.

Reply:
Future PR